ui/package.json

```diff
@@ -28,6 +28,7 @@
     "@types/react-router-dom": "^4.2.3",
     "@types/superagent": "^3.5.7",
     "argo-ui": "git+https://github.com/argoproj/argo-ui.git",
+    "argo-ux": "^1.1.8",
```

Comment:
`argo-ux` increases uncombressed bundle size by 2mb. 5.9mb -> 7.84mb. I've use https://www.npmjs.com/package/webpack-bundle-analyzer  and found two main sources:

* moment and rxjs are bundled twice (1mb). 
*  @fortawesome (1mb)

Can you please make moment and rxjs peer dependencies?  Regarding  @fortawesome - since it increases bundle size that much may be we can just use classes and don't bloat bundle?

Reply:
Will do. Will open another PR in Rollouts to switch from FontAwesome components to classes, and another in Argo UI to do the same